.github/workflows/backend-simple.yml

```
@@ -0,0 +1,56 @@
+      - 'backend/**'
+      - '.github/workflows/backend-simple.yml'
+  pull_request:
+    branches: [ main, develop ]
+    paths:
+      - 'backend/**'
+      - '.github/workflows/backend-simple.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)